### DISCUSSION TITLE
Static section headings in the sidebar

### DISCUSSION THREAD

**Discussion by zackbatist** [quarto-dev/quarto-cli#11692] [Q&A]
### Description

I'm working on a quarto website and I want to have a static section heading in the sidebar. It is possible to emulate this only if I specify an `index.qmd` file, but I do not want a landing page for it either. I just want it to have a few pages nested below a header, with the option for the header to be non-foldable / non-collapsible. I did a comprehensive search but I don't see that this is currently possible.

**Comment by mcanouil** [comment]
Why not use "section" as described in the documentation for sidebar? https://quarto.org/docs/websites/website-navigation.html#side-navigation

**Reply by zackbatist** [reply]
This provides no option to make the section non-collapsible.

**Comment by zackbatist** [comment]
It might also be nice if we could ignore the name of the directory in the url slug.

With the following structure, the slug is `https://example.org/project-documentation/research-protocol.html`.

But it would be great if it could be `https://example.org/research-protocol.html` instead, since the name of the section is just aesthetic in this use case.

```
- section: "Project Documentation"
  contents:
    - project-documentation/research-protocol.qmd
```

**Reply by mcanouil** [reply]
Quarto uses your directory structure to define the URLs.
If you don't want that, then change the structure of your project or use aliases.
https://quarto.org/docs/websites/website-navigation.html#redirects

**Reply by zackbatist** [reply]
Thank you, this is very helpful.
